lib/cache.sh

```diff
@@ -194,3 +194,23 @@ save_custom_cache_directories() {
 
   meta_set "node-custom-cache-dirs" "true"
 }
+
+save_corepack_cache_signature() {
+  local package_manager=${1:-}
```

Comment:
What happens if `pnpm@latest`, `yarn@*`, `npm@8.x` is the `package_manager`? Will we ever invalidate that? Should we use the actual version instead?